### ISSUE TITLE
The application "Ballance" cannot be opened (MacOS)

### ISSUE THREAD

**Issue by Berg085** [imengyu/Ballance#29]
When trying to launch the maxos version (M1), I get this error.  
<img width="264" alt="Screenshot 2024-02-08 at 7 47 52 AM" src="https://github.com/imengyu/Ballance/assets/159450440/a4551efb-eb0b-4ef5-9bdf-ded36b58f63e">

M1 macbook air, running Sonoma 14.3

**Comment by aaasoft** [comment]
Which release file do you use?

**Comment by aaasoft** [comment]
> I tried both 1.0.5 and 1.0.6
> 
> downloaded the 1.0.4 version, and it does the same.
> 
> the 1.0.2 version says its damaged and cannot be opened.

Arch is the point.Macbook M1 use arm64 arch cpu.should use osx-arm64 release file.do you use osx-x64 release file?

**Comment by Berg085** [comment]
> > I tried both 1.0.5 and 1.0.6
> > downloaded the 1.0.4 version, and it does the same.
> > the 1.0.2 version says its damaged and cannot be opened.
> 
> Arch is the point.Macbook M1 use arm64 arch cpu.should use osx-arm64 release file.do you use osx-x64 release file?

Yes, this is trying the osx-arm64 on any version that it's available on.

**Comment by aaasoft** [comment]
> > > I tried both 1.0.5 and 1.0.6
> > > downloaded the 1.0.4 version, and it does the same.
> > > the 1.0.2 version says its damaged and cannot be opened.
> > 
> > 
> > Arch is the point.Macbook M1 use arm64 arch cpu.should use osx-arm64 release file.do you use osx-x64 release file?
> 
> Yes, this is trying the osx-arm64 on any version that it's available on.

Sorry about this.Cas I don't have a mac device,I can't test runing on mac os.The file is published from Unity 2021.Can you install Unity 2021,and have a test?

**Comment by Berg085** [comment]
> > > > I tried both 1.0.5 and 1.0.6
> > > > downloaded the 1.0.4 version, and it does the same.
> > > > the 1.0.2 version says its damaged and cannot be opened.
> > > 
> > > 
> > > Arch is the point.Macbook M1 use arm64 arch cpu.should use osx-arm64 release file.do you use osx-x64 release file?
> > 
> > 
> > Yes, this is trying the osx-arm64 on any version that it's available on.
> 
> Sorry about this.Cas I don't have a mac device,I can't test runing on mac os.The file is published from Unity 2021.Can you install Unity 2021,and have a test?

I'll try, but with the download being pretty big, I'll have to test it some other time.  

I did do some reading, and it seems that other people are experiencing various issues with Unity and Sonoma.  A quick look at the Unity forums shows a lot of threads about issues.
